src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElement.java

```diff
@@ -0,0 +1,20 @@
+package io.github.dftrakesh.cloverrest.model.Order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OrderElement {
+
+    private String id;
+    private String href;
+    private String currency;
+    private String paymentState;
+    private Boolean taxRemoved;
+    private Boolean isVat;
+    private String state;
+    private Long createdTime;
+    private Long modifiedTime;
+    private ListOrderLineItems lineItems;
```

Comment:
change name "lineItems" to orderLineItems

Reply:
changed name "lineItems" to orderLineItems